packages/itinerary-body/src/util.ts

```diff
@@ -157,18 +157,20 @@ export function getPlaceName(
     companies
   );
   if (
+    // Don't ever show this useless OTP default string
+    place.name === "Default vehicle type" ||
     (place.name.match(/-/g) || []).length > 3 ||
     company?.overridePlaceNames
   ) {
-    if (company && intl) {
+    if (intl) {
```

Comment:
Can this if block be merged with the above one?

Reply:
Sure can! Good catch thanks